Mathlib/Algebra/Category/CoalgebraCat/Basic.lean

```diff
@@ -21,10 +24,12 @@ variable (R : Type u) [CommRing R]
 
 /-- The category of `R`-coalgebras. -/
 structure CoalgebraCat extends ModuleCat.{v} R where
-  isCoalgebra : Coalgebra R carrier
+  [isCoalgebra : Coalgebra R carrier]
 
 attribute [instance] CoalgebraCat.isCoalgebra
 
+initialize_simps_projections CoalgebraCat (-isCoalgebra)
```

Comment:
This PR looks good to me, except doesn't this line also remove the `isCoalgebra_comul, isCoalgebra_counit` projections? I think I use those in various other files

Reply:
Removed.